README.adoc

```diff
@@ -1,221 +1,42 @@
-
-=== Prerequisites
-You will need the following tools in your editing environment:
-
-* A bash shell environment (Linux distributions and OS X include these out of the box, for Windows consider http://cygwin.com/[Cygwin])
-* https://www.ruby-lang.org/en/[Ruby]
-* http://www.git-scm.com/[git]
-* A web browser (Firefox, Chrome or Safari) with the http://livereload.com/[LiveReload] extension
-
-With these tools available, first perform a one-time setup:
-
-1. Clone the https://github.com/openshift/openshift-docs[openshift-docs] repo from GitHub:
-+
-----
-$ git clone https://github.com/openshift/openshift-docs.git
-----
-2. From the cloned directory, run a bundle install:
-+
-----
-$ cd openshift-docs
-$ bundle install
-----
-+
-TIP: If you don't have bundler installed, you can get it by running `gem install bundler`
-
-That's it for setup, the next section explains how to run the LiveReload system.
-
-=== Running with LiveReload
-Once you've installed the link:#prerequisites[prerequisites] you can fire up the LiveReload setup as follows:
-
-1. From the `openshift-docs` directory, run a preliminary build:
-+
-----
-$ cd openshift-docs
-$ bundle exec rake build
-----
-2. Now open the generated HTML file in your browser. It will be under `openshift-docs/_preview/<distro>/<branch>` with the same path and filename as the original file. The only difference will be the name ending in '.html' instead of '.adoc'.
-3. Now start up the `guard` utility:
-+
-----
-$ bundle exec guard
-----
-+
-TIP: This utility will run in the terminal where you started it, so you should leave it running off to the side and use other terminals for regular tasks.
-4. Finally, back in your browser, enable the LiveReload plugin in the same tab where the preview file is displayed. You will know this step succeeded if the LiveReload icon changes, and if you see output similar to the following in the terminal where `guard` is running:
-+
-----
-[1] guard(main)> 17:29:22 - INFO - Browser connected.
-----
+All OpenShift documentation is sourced in http://www.methods.co.nz/asciidoc/[AsciiDoc] and transformed into HTML/CSS and other formats through automation that is based on http://asciidoctor.org/[AsciiDoctor].
 
-That's it. Now any changes that you make to the source file will automatically trigger a rebuild of the target HTML file.
+The documentation published from these source files can be viewed at http://docs.openshift.com.
 
-=== Clean Up
-The `.gitignore` file is set up to prevent anything under `_preview` and `_package` from being committed. However, you can reset the environment manually by running:
+== Contributing to OpenShift documentation
+If you are interested in contributing to OpenShift technical documentation, you can view all our link:../contributing_to_docs[resources] that will help you get set up and provide more information. 
 
```

Comment:
@CowboysFan Minor nitpick: should probably be "If you are interested in contributing to <the> OpenShift..."